cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -1188,7 +1192,13 @@ func (r *TrafficOpsReq) StartServices(syncdsUpdate *UpdateStatus, metaData *t3cu
 			log.Errorln("ATS configuration has changed.  The new config will be picked up the next time ATS is started.")
 		} else if serviceNeeds == t3cutil.ServiceNeedsReload {
 			log.Infoln("ATS configuration has changed, Running 'traffic_ctl config reload' now.")
-			if _, _, err := util.ExecCommand(config.TSHome+config.TrafficCtl, "config", "reload"); err != nil {
+			reloadCommand := config.TSHome + config.TrafficCtl
+			reloadArgs := []string{"config", "reload"}
+			if cfg.CacheType == "varnish" {
+				reloadCommand = "varnishreload"
```

Comment:
This seems to be relying on `varnishreload` being in the running user's `$PATH` - with ATS cache servers we typically install everything under `/opt/trafficserver` (not that I think that's a good idea, personally) but it seems like that won't work for varnish caches if everything winds up installed under e.g. `/opt/varnish`. Or, at least, not without some extra work. Is that intentional, or is there some reason why it wouldn't find `varnishreload` in the same directory as the `varnish` binary?

Reply:
It is done that way because Varnish currently is installed under root. So, `varnishreload` is currently installed in user's `$PATH`. It will be a problem indeed if Varnish is installed under different directory. Should Varnish be installed under different directory?